tavern/tomes/file_tree/metadata.yml

```diff
@@ -8,3 +8,7 @@ paramdefs:
   type: string
   label: File path
   placeholder: "/etc/"
+- name: depth
+  type: int
+  label: Recurse Depth
+  placeholder: "-1"
```

Comment:
I'd expand on the -1 in the place holder something like:
"3, or -1 for infinite recursion"